File: Python/Warmup-1/last_digit.py

# this function return true if the two given not-negative int values have the same last digit
def last_digit(a, b):
	bool = False
	s1 = str(a)
	s2 = str(b) 

	if s1[-1] == s2[-1]:
		bool = True

	return bool

print(last_digit(7, 17))
print(last_digit(6, 17))
print(last_digit(3, 113))

File: JavaScript/Warmup-1/lastDigit.js

// given two non-negative int values, return true if they have the same last digit, such as with 27 and 57
const lastDigit = (a, b) => {
    return a % 10 === b % 10 ? true : false;
};

console.log(lastDigit(7, 17));
console.log(lastDigit(6, 17));
console.log(lastDigit(3, 113));
